core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/store/dot-ema-dialog.store.ts

```diff
@@ -309,20 +309,17 @@ export class DotEmaDialogStore extends ComponentStore<EditEmaDialogState> {
     }
 
     private createTranslatePageUrl(page: DotPage, newLanguage: number | string) {
-        const isLive = page.live;
-        const pageLiveInode = page.liveInode;
-        const iNode = page.inode;
-        const stInode = page.stInode;
-
+        const { working, workingInode, inode } = page;
+        const pageInode = working ? workingInode : inode;
         const queryParams = new URLSearchParams({
             p_p_id: 'content',
             p_p_action: '1',
             p_p_state: 'maximized',
             angularCurrentPortlet: 'edit-page',
-            _content_sibbling: isLive ? pageLiveInode : iNode,
+            _content_sibbling: pageInode,
```

Comment:
@rjvelazco should have a default value in case that `pageInode` be `undefined`.

Reply:
@nicobytes  In this context, `pageInode` should never be `undefined`. If you are editing a page, that page must have been created before.

P.S.: We can't have a default `inode` value because an error is generated if we send an inode that is not registered in the database.